### PR TITLE
Fix #3587: Merge topic revision fragments

### LINKED ISSUE #3587: Merge topic_revision_fragment.xml into single xml file
Currently there are 4 versions of `topic_revision_fragment.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by ayaazansari**
Fixes #3587 
Merged the 4 topic revision-fragment xmls (landscape/portrait of phone and tablet )into one revision-fragment .

**Before Commit-**

**Phone**
Screenshot_2021-10-04-22-13-24-239_org oppia android

Screenshot_2021-10-04-22-13-31-952_org oppia android

**Tablet**
por_upstream

land_upstream

**After Commit-**

**Phone**

Screenshot_2021-10-04-22-18-18-403_org oppia android
Screenshot_2021-10-04-22-18-28-212_org oppia android

**Tablet**
por_after_commit
land_after_commit

### CODE REVIEW

**Comment by ayaazansari** [comment]
@rt4914 I'm getting this error when I try to open the fragment in the emulator please help!!
java.lang.IllegalArgumentException: The tag for topic_revision_fragment is invalid. Received: layout-sw600dp-port/topic_revision_fragment_0

File: app/src/main/res/values-land/dimens.xml [hunk 1]
```diff
@@ -307,4 +307,10 @@
   <dimen name="promoted_story_card_layout_margin_start">8dp</dimen>
   <dimen name="promoted_story_card_text_size">14sp</dimen>
   <dimen name="promoted_story_card_margin_top">8dp</dimen>
+
+  <!-- Topic Revision Fragment -->
+  <dimen name="topic_revision_fragment_paddingStart">56dp</dimen>
```
Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml]: Rename this and all other dimens name to `topic_revision_fragment_paddingStart `.
Basically all names should contain only small-characters separated by `_`.

File: app/src/main/res/values-land/dimens.xml [hunk 2]
```diff
@@ -307,4 +307,10 @@
   <dimen name="promoted_story_card_layout_margin_start">8dp</dimen>
   <dimen name="promoted_story_card_text_size">14sp</dimen>
   <dimen name="promoted_story_card_margin_top">8dp</dimen>
+
+  <!-- Topic Revision Fragment -->
```
Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml]: Rename to ` <!-- TopicRevisionFragment -->`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks.

Please add before and after screenshot for this UI.

Also, once you remove tablet related files, make sire you install the app from tablet device, rebuild the app and then install the app again on tablet and it should work correctly.

**Comment by ayaazansari** [comment]
@rt4914 PTAL

File: app/src/main/res/layout/topic_revision_fragment.xml [hunk 3]
```diff
@@ -20,10 +20,10 @@
       android:layout_height="wrap_content"
       android:clipToPadding="false"
       android:overScrollMode="never"
-      android:paddingStart="20dp"
-      android:paddingTop="20dp"
-      android:paddingEnd="20dp"
-      android:paddingBottom="144dp"
+      android:paddingStart ="@dimen/topic_revision_fragment_padding_start"
```
Comment (rt4914) [inline app/src/main/res/layout/topic_revision_fragment.xml]: There is this space before `=`. Please remove it

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by rt4914 — CHANGES_REQUESTED** [review]
One nit change suggested above.

File: app/src/main/res/layout/topic_revision_fragment.xml (hunk 3 above)
Comment (ayaazansari) [reply app/src/main/res/layout/topic_revision_fragment.xml]: @rt4914 done.

**Comment by rt4914** [comment]
@ayaazansari Please merge this latest develop as there are merge conflicts as I had just merged a couple of PRs.

**Comment by ayaazansari** [comment]
@rt4914 done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**